src/mmw/sass/pages/_data-catalog.scss

```diff
@@ -46,9 +69,9 @@
         }
         .data-catalog-search-icon {
             color: $ui-grey;
-            position: relative;
```

Comment:
This was previously set to `relative` to account for the changing height when toggling the date filters.  Using `absolute` keeps it fixed even when the search box itself is pushed down.